utils/test/src/main/kotlin/Utils.kt

```diff
@@ -75,6 +77,22 @@ fun patchExpectedResult(
         .replaceIfNotNull("<REPLACE_URL_PROCESSED>", urlProcessed)
 }
 
+fun patchExpectedResult2(expectedResultFile: File, definitionFile: File): String {
```

Comment:
Can we leave as-is for now, to avoid exceeding the line length in some cases. 
I hope to be able to migrate all occurrences of `patchExpectedResult()`, so that that name could be used later on again.

Reply:
Ok.